sources/packages/web/src/services/http/common/HttpBaseClient.ts

```diff
@@ -78,4 +79,19 @@ export default abstract class HttpBaseClient {
     this.handleRequestError(error);
     throw error;
   }
+
+  // TODO: ONCE ALL URL'S ARE UPDATED, THEN THIS
+  // FUNCTION CAN BE A PRIVATE FUNCTION AND CALLED
+  // BY getCallTyped OR PATCH OR POST FUNCTION
+  // WHEN NEW MODULES ARE ADDED, UPDATE THE CASE ALSO
+  // SO, THAT THIS FUNCTION CAN BE USED TO ADD HOOD TO ALL

```

Comment:
Can we relace the _HOOD_ here for client root also?
Why the comments are all upper case? 😅